lib/cisco_node_utils/interface.rb

```diff
@@ -806,6 +806,61 @@ def feature_lacp_set(val)
       config_set('interface', 'feature_lacp', state: val ? '' : 'no')
     end
 
+    def load_interval_counter_1_delay
+      return nil if @name[/loop/i]
```

Comment:
Check to see if this `return nil` is even needed for the `puppet resource` command.